sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosTemplate.java

```diff
@@ -755,7 +852,7 @@ public <T> Boolean exists(@NonNull CosmosQuery query, @NonNull Class<T> domainTy
 
     /**
      * Delete the DocumentQuery, need to query the domains at first, then delete the item from the result. The cosmos db
-     * Sql API do _NOT_ support DELETE query, we cannot add one DeleteQueryGenerator.
+     * Sql API do _NOT_ support DELETE query, we cannot add one DeleteQueryGenerator. Uses bulk if possible.
```

Comment:
Thanks @trande4884  for adding this documentation. Can we put this documentation on `CosmosOperations` and `SimpleCosmosRepository` as well so it becomes more visible.